### DISCUSSION TITLE
Definition of ‘Act of Deceptive Communication’ too narrow?

### DISCUSSION THREAD

**Discussion by gregfowlerphd** [CommonCoreOntology/CommonCoreOntologies#907] [Ontology Terms and Class Schema]
The definition reads:

> Ac Act of Communication intended to mislead the audience by distortion or falsification of evidence and induce a reaction that is prejudicial to the interests of the audience.

While the first condition seems basically right, I worry about the second. Consider the case of the Pacific Northwest tree octopus hoax. In positing the website, the creator might well have intended to mislead the audience. Let’s assume he did. It’s doubtful, however, that he intended to induce a prejudicial reaction of the relevant sort. (How could believing in such an octopus be harmful to the audience’s interests?) Nonetheless, it would seem he engaged in an act of deceptive communication.

(Minor second issue: The definition begins with ‘Ac’ rather than ‘An’.)

**Comment by cameronmore** [comment]
Perhaps there is a deceptive act of communication and a (new) subclass might just include the second condition.

**Comment by gregfowlerphd** [comment]
That might work.

As far as the judgment that there's an act of deceptive communication goes, I'm basing it on the assumption that the words in the label 'act of deceptive communication' are to be interpreted as having their standard English meanings (i.e., that 'act of deceptive communication' isn't supposed to be a technical term). If that assumption's correct, then an act of deceptive communication would simply be an act of communication that 'causes [someone] to accept as true or valid what is false or invalid'. (See the Merriam-Webster entry for 'deceive' here: https://www.merriam-webster.com/dictionary/deceive.)

**Comment by cameronmore** [comment]
I think the second condition ("induce a reaction that is prejudicial to the interests of the audience") should be added as a note (rdfs:comment or skos:scopeNote) to this class and removed from the definition. That condition may be an outcome, or intended outcome, but there are cases where it is not as you pointed out. Can you make the change and pr?

**Comment by gregfowlerphd** [comment]
@cameronmore: I can do that. However, I'm a bit worried because the definition, revised along these lines, will still contain the word 'audience', which @swartik had concerns about. (See his comment on my #301.) Might still be worth revising at this point, since doing so would fix one problem with the definition and concerns about 'audience' can be addressed later. What do you think?

**Comment by cameronmore** [comment]
we can change to recipient as well since that term is mentioned in the object properties receives/has_recipient

**Comment by swartik** [comment]
One way to address my concerns is to declare a specific dictionary as the definitional source for words used in CCO annotations. I check the Oxford English Dictionary first, but I know many, particularly on this side of the Atlantic, favor Merriam-Webster. If the CCO specification stated which of these (or some other dictionary) to consult, we could save a lot of back-and-forth discussion about what a word means in a definition.

**Comment by cameronmore** [comment]
If 'recipient' addresses this term's issue, I think we should commit the change and open a new issue or discussion post to explore that idea.

**Comment by gregfowlerphd** [comment]
@cameronmore: Your suggestion to replace 'audience' with 'recipient' is a good one, and I think it might help with Steve's general worry.

I do think we'll have to do some other tweaks here though. If we simply remove the second condition from the ADC definition and do the 'audience'/'recipient' switch, we end up with the following:

> **Act of Deceptive Communication:** An Act of Communication intended to mislead the recipient by distortion or falsification of evidence.

But '_the_ recipient' suggests uniqueness--i.e., that an ADC can have only one recipient--which is incorrect.

In response, one might propose replacing 'the recipient' with 'its recipient(s)'. However, an ADC needn't be intended to mislead _all_ of its recipients (since some of its recipients might be in on the lie), so this doesn't work.

I think it might work to replace 'the recipient' with 'its recipient or one of its recipients' or with 'at least one of its recipients'. Alternatively, perhaps we could get away with not mentioning an audience or recipient at all here, as follows:

> **Act of Deceptive Communication:** An Act of Communication intended to mislead by distortion or falsification of evidence.

Although I'm not sure that's explicit enough.

What do you think?

**Comment by cameronmore** [comment]
@gregfowlerphd Being that the parent class already has a notion of transferring information from one Agent to another ("A Planned Act in which some Information Content Entity is transferred from some Agent to Another"), we may drop it altogether as you suggest.

**Comment by gregfowlerphd** [comment]
@cameronmore: PR created.

As far as using 'recipient' to address Steve's general worry about 'audience' (as well as my worry about a number of the definitions using 'hearer'), do you think it's worth opening a new issue or discussion post about that proposal? Or should we instead put it in a comment on discussion post 369 (the discussion post my #301 was converted into)?

**Comment by cameronmore** [comment]
I think a new discussion post is best for the broader question @swartik raises

**Comment by johnbeve** [comment]
This thread highlights why prejudice to the audience should not be built into the definition, that audience/recipient language creates unnecessary complications, and that misleading by falsification is an important, but not the only way to deceive. My proposal below sidesteps the earlier worries by shifting attention to the underlying structure common to deception. My preferred characterization: 

> Act of Deceptive Communication =Def An Act of Communication during which an agent A transfers information to at least one agent and in doing so aims to conceal whether that information is true or not.

Examples of deception are not best understood as falsification of evidence. Deception often consists in presenting content in such a way that the recipient is misled about whether it is true, e.g. selective disclosure, omission, etc. My proposal captures this but also preserves standard lying cases, which can be reflected by a subclass: 

> Act of Lying =Def An Act of Deceptive Communication during which an agent A states information p to agent B, A believes p is false, and A aims to make B believe p is true.

Bernie Madoff stated information he believed to be false and aimed to bring about false belief in others, i.e. he lied. 

Lying is only one species of deceptive communication. Consider, Sam has been interrogated by law enforcement for several hours. He is a suspected car thief and is being questioned regarding his alibi. During the interrogation, Sam at one point claims he was at home at 9pm and at another claims he was at a bar at 9am. When this is pointed out, Sam refuses to respond. Sam is not trying to state information he believes is false and get the interrogator to believe it true, but to conceal whether relevant information is true or not. That is deceptive, but not strictly speaking lying. 

As another example, suppose Mary is a known pathological liar. Mary and Bob are to meet in Room 1 on Monday, but Mary learns they must meet in Room 2 instead. Mary knows Bob believes she is a liar, so Mary tells Bob they are to meet in Room 2. Because she knows Bob takes her to be a liar, she says something true precisely in order to mislead him. This is deceptive, yet it is not a lie, since she does not believe the communicated content is false.
